tools/flang1/flang1exe/accpp.c

```diff
@@ -701,6 +701,20 @@ print_and_check(FILE *ff, char *str, char end_c)
     fputc(end_c, ff);
 }
 
+extern bool g_list_macros;
+void acc_display_macros()
+{
+  for (unsigned int i = 1; i<next_hash; ++i) {
```

Comment:
this is C code, so although today's compilers understand this C++ way of defining variables, you should follow C rules.
should be:
```
{
  unsigned int i;

  for (i = 1U; i < next_hash; ++i) {
```